pyprotista/parsers/misc.py

```diff
@@ -1,113 +1,83 @@
     isotope_masses = None
     isotope_probs = None
-    try:
-        get_composition_and_mass_and_accuracy.cc.use(sequence=seq, modifications=mods)
-        composition = get_composition_and_mass_and_accuracy.cc.hill_notation_unimod()
-        replaced_composition = composition
-        c12_mass = get_composition_and_mass_and_accuracy.cc.mass()
-        static_isotopes = re.findall(r"(?<=\))(\d+)(\w+)(?:\()(\d+)", composition)
-        if len(static_isotopes) != 0:
-            atom_counts = []
-            isotope_masses = []
-            for isotope in static_isotopes:
-                mass, element, number = isotope
-                replaced_composition = replaced_composition.replace(
-                    f"{mass}{element}({number})", ""
-                )
-                isotope_masses.append(
+    replaced_composition = composition
+    static_isotopes = re.findall(r"(?<=\))(\d+)(\w+)(?:\()(\d+)", composition)
+    if len(static_isotopes) != 0:
+        atom_counts = []
+        isotope_masses = []
+        for isotope in static_isotopes:
+            mass, element, number = isotope
+            replaced_composition = replaced_composition.replace(
+                f"{mass}{element}({number})", ""
+            )
+            isotope_masses.append(
+                [
                     [
-                        [
-                            m
-                            for m in symbol_to_masses[element]
-                            if str(round(m)).startswith(mass)
-                        ][0]
-                    ]
-                )
-                atom_counts.append(int(number))
-            isotope_probs = len(atom_counts) * [[1.0]]
-        if replaced_composition is not None:
-            formula = replaced_composition
-        else:
-            formula = composition
-        formula = formula.replace("(", "").replace(")", "")
-        isotopologue_masses = list(
-            iso.IsoThreshold(
-                formula=formula,
-                threshold=0.02,
-                charge=1,
-                get_confs=True,
-                atomCounts=atom_counts,
-                isotopeMasses=isotope_masses,
-                isotopeProbabilities=isotope_probs,
-            ).masses
-        )
-        isotopologue_mzs = [
-            get_composition_and_mass_and_accuracy.calc_mz(mass, charge)
-            for mass in isotopologue_masses
-        ]
-        # Report only most accurate mass
-        isotopologue_mz = min(
-            isotopologue_mzs,
-            key=lambda x: abs(exp_mz - x),
-        )
-        isotopologue_acc = (exp_mz - isotopologue_mz) / isotopologue_mz * 1e6
-    except (KeyError, Exception):
-        logger.warning(f"Could not calculate mz for {seq}#{mods}")
-        pass
-    return composition, c12_mass, isotopologue_acc
+                        m
+                        for m in symbol_to_masses[element]
+                        if str(round(m)).startswith(mass)
+                    ][0]
```

Comment:
Reads awkward - what does it do? I guess 14N ? I would suggest a different data structure ... Maybe wrong - phone / train / again